packages/server/src/__tests__/standalone/integration.test.ts

```diff
@@ -0,0 +1,21 @@
+import { standaloneServer } from '../../standalone';
+import type { BaseContext } from '../../externalTypes';
+import type {
+  CreateServerForIntegrationTestsOptions,
+  CreateServerForIntegrationTestsResult,
+} from '../integration';
+import { defineIntegrationTestSuite } from '../integration';
+import type { ApolloServerOptions } from '../../types';
+import { ApolloServer } from '../../ApolloServer';
+
+defineIntegrationTestSuite(async function (
+  serverOptions: ApolloServerOptions<BaseContext>,
+  testOptions?: CreateServerForIntegrationTestsOptions,
+): Promise<CreateServerForIntegrationTestsResult> {
+  const opts = testOptions ? { context: testOptions.context } : undefined;
+  const server = new ApolloServer(serverOptions);
+  const standaloneServerInstance = standaloneServer(server, opts);
+  await standaloneServerInstance.listen({ port: 0 });
+
+  return { server, httpServer: standaloneServerInstance['httpServer'] };
```

Comment:
It might make sense to update the integration test's API to take an URL instead of an http.Server (are there any uses of the httpServer that can't be replaced by an URL? supertest can). That way we could support integrations that somehow don't literally use an http.Server (maybe some serverless things might work that way?) as long as they can serve at an URL. I think this could be a follow-up PR though!